packages/web-app-admin-settings/src/components/Users/GroupsModal.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <oc-modal
+    :title="title"
+    :message="message"
```

Comment:
The message seems to be unused when you overwrite the content slot. 🙈 💩  i.e. the message doesn't appear at all in the modal.